WordPress/Classes/ViewRelated/Voice/VoiceToContentViewModel.swift

```diff
@@ -260,18 +268,25 @@ final class VoiceToContentViewModel: NSObject, ObservableObject, AVAudioRecorder
 }
 
 private enum VoiceToContentError: Error, LocalizedError {
+    case generic
     case cantUnderstandRequest
 
     var errorDescription: String? {
         switch self {
+        case .generic: return Strings.errorMessageGeneric
         case .cantUnderstandRequest: return Strings.errorMessageCantUnderstandRequest
         }
     }
 }
 
+private enum Constants {
+    static let recordingTimeLimit: TimeInterval = 5 * 60 // 5 Minutes
+    static let recordingTimeLimitFormatted = "05:00" // 5 Minutes
```

Comment:
(nit) I'd suggest passing this through the formatter and using `recordingTimeLimit`

Reply:
Fixed! [16f8171](https://github.com/wordpress-mobile/WordPress-iOS/pull/23279/commits/16f817117cf28bb9aabb146d19f998f73bb55d60)